test/models/report_test.rb

```diff
@@ -1,9 +1,60 @@
 # frozen_string_literal: true
 
 require 'test_helper'
+require 'debug'
 
 class ReportTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  setup do
+    FactoryBot.create(:report, id: 10)
+    FactoryBot.create(:report, id: 11)
+    FactoryBot.create(:report, id: 12)
+    FactoryBot.create(:report, id: 13)
+  end
+
+  test '#editable? should return true when target user tries to edit a report created by target user' do
+    target_user = FactoryBot.create(:user)
+    report = FactoryBot.create(:report, user: target_user)
+
+    assert report.editable?(target_user)
+  end
+
+  test '#editable? should return false when target user tries to edit a report created by another user' do
+    target_user = FactoryBot.create(:user)
+    another_user = User.create(email: 'bob@example.com', name: 'Bob', password: 'password')
```

Comment:
```suggestion
    another_user = FactoryBot.create(:user)
```

こっちもFactoryBotを使うといいのでは、と思いました。

Reply:
`another_user`をFactorybotで定義して使うようにしました。